secure-releases.md

```diff
@@ -0,0 +1,60 @@
+
+## CVE Management
+
+### Security Policy
+
+Your software project will, likely inevitably, have a security vulnerability. When (not if) that happens, you want to make sure that whoever discovers it has a way to report it to you privately - this is called "responsible disclosure" or "coordinated disclosure". So, the first important step is to *have a security policy*.
+
+On Github, this is always located at `$REPO_URL/security/policy` ([example](https://github.com/nvm-sh/nvm/security/policy)). This page can be populated from a number of file locations: `SECURITY.md` or `.github/SECURITY.md` in the repo itself, or, one of the same files in your user or organization’s `.github` repository. If you're not on Github, then look for a similar platform-specific convention, and if you find none, a top-level `SECURITY.md` is safe.
+
+In this policy, you'll want to mention a private reporting channel. This could be an email address, a service like HackerOne, etc.
+
+#### Programmatic Private Vulnerability Reporting
+
+Github and Gitlab support a Private Vulnerability Reporting (PVR) feature and API. This allows reporters to use the platform itself, programmatically if needed. It is strongly recommended to enable this feature, if it's not already on by default.
+
+### Audit Dependencies
+
+You'll want to ensure all of your dependencies - direct and transitive - are audited on every commit. This can be achieved locally and in CI with `npm audit`, if your project has a lockfile, or https://npmjs.com/aud for packages without a lockfile.
+
+In addition - not instead - you may want to use services like Dependabot, Renovate, [Tidelift](https://tidelift.com), [Socket.dev](https://socket.dev), etc to automatically detect and warn you about issues with your dependency graph.
+
+If your package does not have a build process - meaning, all of the artifacts you publish are committed to version control and hand-written - then you may only need to audit your runtime dependencies (`dependencies`, `peerDependencies`) and not `devDependendies`. However, if you do have a build process, whichever dev deps that generate code need to be audited as well.
+
+### Handling Reports
+
+First, be highly responsive. Ideally, you can respond to a reporter within 24 hours or 1 business day. Your security policy should explicitly describe your expected response time - especially if it's longer than this.
```

Comment:
I'm a bit torn on this. Making people feel bad if they don't respond within a day is great for people using the package for free but not so great for maintainers.  I personally think what is important is to state what kind of response time to expect and that it reflect what the maintainer will be realisticly be able to achieve.

If we want a higher bar for this for OpenJS projects, then we might want to include that in one of the entry/application checklists.

Reply:
or I guess offer some support in achieving a fast turn around.

Reply:
I used "ideally" so that they can set any time they like, as long as it's in the policy.

How would you suggest I rephrase it where I'm still establishing a norm of responsiveness?

Reply:
How about this

```
It is important to set expectations for how quickly you will be able to respond to reporters by documenting this in your security policy. It is more imporant to set a realistic timeframe and meet it than setting a short timeframe that you cannot meet. Once you set the timeframe in your security policy work to meet that consistently.

Reply:
If this guide is mean to be specific to OpenJS projects versus a general guide we could then add a requirement along the lines of something like "As an OpenJS foundation project the requirement is that the response timeframe be 1 week or less" or whatever the requirement is.

Reply:
It's meant to cover both foundation projects and general JS ecosystem projects.

I don't think there should be any requirement here, necessarily - but I think a business day is a realistic default for security reports.

Reply:
Even within a large project like Node.js we don't make 1 day. This is what we've ended up with:

Normally your report will be acknowledged within 5 days, and you'll receive a more detailed response to your report within 10 days indicating the next steps in handling your submission. These timelines may extend when our triage volunteers are away on holiday, particularly at the end of the year.

Reply:
I would expect it's *harder* in a large project to have faster response times - but most open source projects are small, and it should be pretty easy to respond within a day.

I absolutely don't want to shame or preclude longer response times, but 5-10 days would be a pretty long time imo for a small project.

Reply:
Assuming somebody has time to check their project every single day when its not their paid job seems a bit much to ask, but at this point I've shared my opinion and I'll bow out of the dicussion to let whatever the consensus is be used.